rust/windows-client/src-tauri/src/client.rs

```diff
@@ -0,0 +1,69 @@
+use anyhow::Result;
+use clap::Parser;
+use cli::CliCommands as Cmd;
+
+mod cli;
+mod debug_commands;
+mod device_id;
+#[cfg(target_family = "unix")]
+mod gui {
+    use super::*;
+
+    pub enum ControllerRequest {
+        SignIn,
+    }
+
+    pub(crate) fn run(_: GuiParams) -> Result<()> {
+        // The Ubuntu CI runner doesn't have gdk and some other Tauri deps installed, so it fails unless we stub out the GUI.
+        panic!("The Tauri GUI isn't implemented for Linux.");
+    }
+}
+#[cfg(target_os = "windows")]
+mod gui;
+mod local_webserver;
+// Relies on some types from Tauri
+#[cfg(target_os = "windows")]
+mod settings;
+
+/// Prevents a problem where changing the args to `gui::run` breaks static analysis on non-Windows targets, where the gui is stubbed out
+#[allow(dead_code)]
+pub(crate) struct GuiParams {
+    /// The URL of an incoming deep link from a web browser
+    deep_link: Option<String>,
+    /// True if we should slow down I/O operations to test how the GUI handles slow I/O
+    inject_faults: bool,
+}
+
+/// Newtype for our per-user directory in AppData, e.g.
+/// `C:/Users/$USER/AppData/Local/dev.firezone.client`
+pub(crate) struct AppLocalDataDir(std::path::PathBuf);
+
+pub(crate) fn run() -> Result<()> {
+    // Special case for app link URIs
+    if let Some(arg) = std::env::args().nth(1) {
+        let scheme = format!("{DEEP_LINK_SCHEME}://");
+        if arg.starts_with(&scheme) {
+            return gui::run(GuiParams {
+                deep_link: Some(arg),
+                inject_faults: false,
+            });
+        }
+    }
```

Comment:
we can just hardcode the deep link scheme

Reply:
It's also used over here when we register the deep link scheme https://github.com/firezone/firezone/blob/8a82d3fddcc4c422833f0c78cad48ece88ab8329/rust/windows-client/src-tauri/src/client/gui.rs#L108